### PR TITLE
Changelog for 1.8.2.

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,73 +1,44 @@
...
 * Fix string elements to render as code in the IntelliSense configuration UI. [PR #8271](https://github.com/microsoft/vscode-cpptools/pull/8271)
 * Fix IntelliSense process crash on AMD Ryzen 3000 series processors without updated drivers. [#8312](https://github.com/microsoft/vscode-cpptools/issues/8312)
 * Fix bug with `wmic` not being recognized during Windows attach debugging. [#8328](https://github.com/microsoft/vscode-cpptools/issues/8328)
 * Fix Go to Type Definition on pointer types. [#8337](https://github.com/microsoft/vscode-cpptools/issues/8337)
-* Fix MS extensions not being enabled by default for Cygwin. [#8353](https://github.com/microsoft/vscode-cpptools/issues/8353)
 * Fix a "Cannot read property" error during deactivation if the language service wasn't fully activated. [#8354](https://github.com/microsoft/vscode-cpptools/issues/8354)
 * Fix an issue in which the language id for header files were not updated to match the source file of its TU. [#8381](https://github.com/microsoft/vscode-cpptools/issues/8381)
+* Fix some IntelliSense bugs. [#8412](https://github.com/microsoft/vscode-cpptools/issues/8412)
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: References to 8486, 8677, and 8701 were removed. Were those insiders regressions that users on 1.7.1 would not have seen? Is that why they're not listed here?

**Review by bobbrow — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
Might add https://github.com/microsoft/vscode-cpptools/issues/8470 still...

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: @bobbrow However, I added a few more IntelliSense (and other) fixes that were missing from our previous changelogs.

**Review by bobbrow — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
> Might add #8470 still...

It moved to 1.9

**Pull request merged**